JetMETCorrections/Modules/plugins/QGLikelihoodESProducer.cc

```diff
@@ -0,0 +1,58 @@
+/*
+ Description: ESProducer to get the quark-gluon likelihood object "QGLikelihoodObject"
+              from record "QGLikelihoodRcd". 
+
+ Implementation:
+     Completely trivial, simply returns the QGLikelihoodObject to the user. There is only
+     one QGLikelihoodObject object in each record. 
+*/
+// Original Author:  Salvatore Rappoccio
+//         Created:  Thu, 13 Mar 2014 15:02:39 GMT
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/Framework/interface/EventSetupRecordIntervalFinder.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/Utilities/interface/do_nothing_deleter.h"
+
+#include "CondFormats/JetMETObjects/interface/QGLikelihoodObject.h"
+#include "CondFormats/DataRecord/interface/QGLikelihoodRcd.h"
+
+class QGLikelihoodESProducer : public edm::ESProducer {
+public:
+  QGLikelihoodESProducer(const edm::ParameterSet &);
+  ~QGLikelihoodESProducer() override{};
+
+  std::shared_ptr<const QGLikelihoodObject> produce(const QGLikelihoodRcd &);
+  void setIntervalFor(const edm::eventsetup::EventSetupRecordKey &, const edm::IOVSyncValue &, edm::ValidityInterval &);
```

Comment:
@Dr15Jones Does the `setIntervalFor()` acqually do anything when the ESProducer does not inherit from `edm::EventSetupRecordIntervalFinder`?

Reply:
It won’t be called.

Reply:
Thanks, I'll remove those